docs/self-managed/identity/troubleshooting/index.md

```diff
@@ -0,0 +1,45 @@
+---
+id: index
+title: "Troubleshooting Identity"
+sidebar_label: "Overview"
+---
+
+## Issue: Identity pod crashloops/continually restarts
+
+If the Identity pod crashloops it is likely that there is an issue with the connection to Keycloak.
```

Comment:
```suggestion
If the Identity pod crashloops, or the pod continually restarts, it is likely that there is an issue with the connection to Keycloak.
```

Is "crashloop" specific to Kubernetes? Again, nit pick, but I would add this to the text (even though its in the header). It's a pretty intuitive word for me, but I'm not sure if it would translate well for non-native English speakers or devs outside of the Kubernetes space.

Reply:
Good question, in this instance its a specific error status within Kubernetes `CrashLoopBackOff` but I agree that the general term `crashloops` potentially does not translate well.

I'll add in the text as it covers all cases 👍🏻

Reply:
Thanks! 

FYI @christinaausley "crashloops" might be a good term for you to research. I'm seeing it online as CrashLoops, crash loop, crashloops, etc. Singular, plural, various casing and spacing. Like Ben mentions, it's related to CrashLoopBackOff.